deker/ABC/base_factory.py

```diff
@@ -52,8 +52,11 @@ def __del__(self) -> None:
     @abstractmethod
     def close(self) -> None:
         """Close own executor."""
-        if self.executor and self._own_executor:
-            self.executor.shutdown(wait=True, cancel_futures=False)
+        try:
+            if self.executor and self._own_executor:
+                self.executor.shutdown(wait=True, cancel_futures=False)
+        except Exception:
```

Comment:
There should be specific Exception. We still want to fail on unexpected things.

Reply:
[52b8348](https://github.com/openweathermap/deker/pull/15/commits/52b8348e12680f1d16548cbd43d8635fe2f109fb)